Performance/Performance112.md

```diff
@@ -28,14 +26,13 @@ We have partnered with BisectHosting this is a exciting step for us. All money e
 | [FastWorkbench](https://www.curseforge.com/minecraft/mc-mods/fastworkbench) | None | This is a mod aimed at improving performance of all crafting-related functions. | Shadows_of_Fire | Server (Must be installed in client as well) | none |
 | [FoamFix](https://www.curseforge.com/minecraft/mc-mods/foamfix-optimization-mod) | None | FoamFix is a mod designed to optimize post-1.7.10 modded Minecraft using simple, targeted optimizations | asiekierka | Both | none |
 | [Get It Together, Drops!](https://www.curseforge.com/minecraft/mc-mods/get-it-together-drops) | None | This mod adds two configuration options to control how dropped items combine on the ground. This can significantly improve performance in areas with lots of dropped items | bl4ckscor3 | Server | none |
-| [Multithreaded Noise](https://www.curseforge.com/minecraft/mc-mods/multithreaded-noise) | Unknown | This mod makes overworld terrain generate faster by replacing the three 3D noise generators in the overworld with multithreaded versions. | bloodnbonesgaming | Server | Alpha (1) |
 | [Nothirium](https://www.curseforge.com/minecraft/mc-mods/nothirium) | Unknown | Nothirium is a client-side only mod that changes Minecraft's chunk rendering engine to use modern OpenGL which greatly increases performance. This mod is similar to Sodium but this is not a port of Sodium. | meldexun | Client | none |
 | [Particle Culling](https://www.curseforge.com/minecraft/mc-mods/particle-culling) | None | This mod is a client-side performance enhancing mod, which stops particles that are outside the player's view from rendering | bl4ckscor3 | Client | none |
 | [Performant](https://www.curseforge.com/minecraft/mc-mods/performant) | Bewitchment | Lightweight mod project which hugely improves performance and blocklag. | someaddon | Server | Incompatible (2) |
 | [Phosphor](https://www.curseforge.com/minecraft/mc-mods/phosphor-forge) | Mods using outdated mixin versions[¹](/README.md#-should-work-if-mixin-0-7-0-8-compatibility-is-applied) | Phosphor is a Minecraft mod which works to optimize one of game's most inefficient areas, the lighting engine | jellysquid_ | Both | none |
 | [PortalSuperCache](https://github.com/LucunJi/PortalSuperCacheForge112/releases) | Unknown | When using this mod, your game will search portal blocks at a high speed on teleport. | LucunJi | Server | Alpha (?) |
+| [Universal Tweaks](https://www.curseforge.com/minecraft/mc-mods/universal-tweaks) | Mods it replaces, mods using outdated mixin versions[¹](/README.md#-should-work-if-mixin-0-7-0-8-compatibility-is-applied)| A crap ton of performance and bugfix mods in one | ACGaming56 | Both | none |
```

Comment:
"Mods it replaces" is generic, and should list at least a few mod names or (as an example) "Mods that affect/modify Mob AI"

Reply:
There is a full list in the mod changelog.